dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DataverseSession.java

```diff
@@ -94,6 +94,10 @@ public String getLocaleTitle() {
     public int getFilesPerPage() {
         return filesPerPage;
     }
+    
+    public boolean canEditDashboard() {
```

Comment:
Should be moved to the LOGIC section below.

Reply:
done